app/src/main/java/com/test/lolinformation/ui/gameplay/detailchamp/DetailChampionViewModel.kt

```diff
@@ -1,7 +1,9 @@
 package com.test.lolinformation.ui.gameplay.detailchamp
 
+import androidx.lifecycle.MutableLiveData
+import com.test.lolinformation.data.local.model.Champion
 import com.test.lolinformation.ui.base.BaseViewModel
 
 class DetailChampionViewModel : BaseViewModel() {
-
+    val champion = MutableLiveData<Champion>()
```

Comment:
Không nên gán value trực tiếp từ View mà nên set qua overhead.
```suggestion
    private val _champion = MutableLiveData<Champion>()
    val champions: LiveData<Champions> get() = _champion
    
    fun setChampionValue(value: Champion) {
        _champion.value = value
    }
```